yt_dlp/extractor/bbc.py

```diff
@@ -1128,6 +1138,7 @@ def _real_extract(self, url):
                     'uploader_id': network.get('id'),
                     'formats': formats,
                     'subtitles': subtitles,
+                    'chapters': tracklist,
```

Comment:
You could inline and make it completely non-fatal by doing this:
```suggestion
                    'chapters': traverse_obj(preload_state, (
                        'tracklist', 'tracks', lambda _, v: float_or_none(v['offset']['start']), {
                            'title': ('titles', {lambda x: join_nonempty(
                                'primary', 'secondary', 'tertiary', delim=' - ', from_dict=x)}),
                            'start_time': ('offset', 'start', {float_or_none}),
                            'end_time': ('offset', 'end', {float_or_none}),
                        })) or None,
```